src/XCCDF/public/xccdf_session.h

```diff
@@ -42,6 +42,19 @@
 struct xccdf_session;
 
 /**
+ * Loading flags for XCCDF session
+ * @memberof xccdf_session
+ * The flags set which components will be loaded in XCCDF session.
+ */
+typedef enum {
+	XCCDF_SESSION_LOAD_NONE = 0,
```

Comment:
I'm sorry, but now I'm confused. Should it be non-zero value? If yes, should it be called XCCDF_SESSION_LOAD_XCCDF? Or do you just suggest keeping this and writing a comment there?

Tbh I can't find a meaningful example of creating an XCCDF session without loading an XCCDF document 😬